c/parallel/test/test_merge_sort.cpp

```diff
@@ -185,8 +185,8 @@ C2H_TEST("DeviceMergeSort:SortPairsCopy works with custom types", "[merge_sort]"
   operation_t op         = make_operation(
     "op",
     "struct key_pair { short a; size_t b; };\n"
-            "extern \"C\" __device__ bool op(key_pair lhs, key_pair rhs) {\n"
-            "  return lhs.a == rhs.a ? lhs.b < rhs.b : lhs.a < rhs.a;\n"
+            "extern \"C\" __device__ void op(key_pair* lhs, key_pair* rhs, bool* out) {\n"
+            "  *out = lhs->a == rhs->a ? lhs->b < rhs->b : lhs->a < rhs->a;\n"
```

Comment:
Lines 188-189 are indented related to line 187 in `test_merge_sort.cpp`. It is not necessary, and since we are changing these lines, it would be nice to remove that indentation too.

Reply:
I'm not sure why but clang-format wants to restore that indentation..